herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java

```diff
@@ -1180,4 +1182,115 @@ public LocalClientScanResultSetImpl executeScan(String tableSpace, String query,
         }
     }
 
+    public List<DMLResult> executeUpdates(String tableSpace, String query, long transactionId, boolean returnValues, List<List<Object>> originalBatch) throws HDBException {
+        if (query == null) {
+            throw new HDBException("bad query null");
+        }
+
+        List<List<Object>> batch = new ArrayList<>(originalBatch.size());
+        for (int i = 0; i < originalBatch.size(); i++) {
+            batch.add(PduCodec.normalizeParametersList(originalBatch.get(i)));
+        }
+        try {
+
+            List<TranslatedQuery> queries = new ArrayList<>();
+            for (int i = 0; i < batch.size(); i++) {
+                List<Object> parameters = batch.get(i);
+                TranslatedQuery translatedQuery = server
+                        .getManager()
+                        .getPlanner().translate(tableSpace, query,
+                                parameters, false, true, returnValues, -1);
+                queries.add(translatedQuery);
+            }
+
+            List<Long> updateCounts = new CopyOnWriteArrayList<>();
+            List<Map<RawString, Object>> otherDatas = new CopyOnWriteArrayList<>();
```

Comment:
typo "otherData"

Reply:
this is not a typo, this is a list of "otherData", so "otherDatas" sounds good to me, do you have a suggestion ?